LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2004-2016 Castle Project - http://www.castleproject.org/
```

Comment:
Don't worry about my second sentence, just rambling really, wondering if this is a recent NuGet regression.

Happy to go which ever way you'd like, I just thought the history would be cleaner with a "rename".